doc/UserGuide.md

```diff
@@ -134,3 +134,8 @@ Example:
 >
 > When running the program inside IntelliJ, you can set command line parameters 
   before running the program.
+
+#### Viewing the file size of the storage file (default: `addressbook.xml`)
```

Comment:
This feature actually make sense, if you choose an appropriate user for the product.